homeassistant/components/openweathermap/sensor.py

```diff
@@ -166,6 +167,11 @@
         key=ATTR_API_FORECAST_CONDITION,
         name="Condition",
     ),
+    SensorEntityDescription(
```

Comment:
- Missing device class?
- I'm not sure what this has to do with forecasting as per PR description/title?